### PR TITLE
Add Spanish translation for linux.md

### PR DESCRIPTION
**Pull request by alemohamad**
Added Spanish translation for linux.md.

### CODE REVIEW

**Review by jacostaf10 — APPROVED** [review]
LGTM, natively correct

File: docs/install/linux.es.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Instalación en Linux
+
+Para usar Vapor, necesitas Swift 5.2 o superior. Se puede instalar usando las opciones disponibles en [Swift.org](https://swift.org/download/).
```
Comment (0xTim) [inline docs/install/linux.es.md:3]:
```suggestion
Para usar Vapor, necesitas Swift 5.6 o superior. Se puede instalar usando las opciones disponibles en [Swift.org](https://swift.org/download/).
```
Nosotros ahora requerimos 5.6 o superior

File: docs/install/linux.es.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Distribuciones y Versiones Soportadas
+
+Vapor admite las mismas versiones de distribución de Linux que adminte Swift 5.2 o versiones más recientes.
```
Comment (0xTim) [inline docs/install/linux.es.md:7]:
```suggestion
Vapor admite las mismas versiones de distribución de Linux que adminte Swift 5.6 o versiones más recientes.
```

File: docs/install/linux.es.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+!!! nota
+    Las versiones soportadas que se enumeran a continuación pueden quedar obsoletas en cualquier momento. Puedes comprobar qué sistemas operativos son oficialmente compatibles en la página [Swift Releases](https://swift.org/download/#releases).
+
+|Distribución|Versión|Versión de Swift|
+|-|-|-|
+|Ubuntu|16.04, 18.04|>= 5.2|
```
Comment (0xTim) [inline docs/install/linux.es.md:14]: 16.04 y 18.04 no son compatibles

File: docs/install/linux.es.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+|Ubuntu|20.04|>= 5.2.4|
```
Comment (0xTim) [inline docs/install/linux.es.md:15]:
```suggestion
|Ubuntu|20.04|>= 5.6|
```

File: docs/install/linux.es.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+|Fedora|>= 30|>= 5.2|
```
Comment (0xTim) [inline docs/install/linux.es.md:16]:
```suggestion
|Fedora|>= 30|>= 5.6|
```

File: docs/install/linux.es.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+|CentOS|8|>= 5.2.4|
```
Comment (0xTim) [inline docs/install/linux.es.md:17]:
```suggestion
|CentOS|8|>= 5.6|
```

File: docs/install/linux.es.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+|Amazon Linux|2|>= 5.2.4|
```
Comment (0xTim) [inline docs/install/linux.es.md:18]:
```suggestion
|Amazon Linux|2|>= 5.6|
```

**Review by 0xTim — CHANGES_REQUESTED** [review]

**Comment by alemohamad** [comment]
@0xTim thanks for the change in the values! Should we update all of them in the other translations (even English)? Maybe I can do this in this PR. Please let me know.

File: docs/install/linux.es.md (hunk 3 above)
Comment (alemohamad) [reply docs/install/linux.es.md:14]: @0xTim Should we change them to another Ubuntu version(s)?

**Comment by 0xTim** [comment]
@alemohamad Yeah if you can do them in this PR that would be great!

**Comment by alemohamad** [comment]
@0xTim updated values in another translations. 😉

**Review by 0xTim — APPROVED** [review]
Great thank you!

**Pull request merged**